src/styles/contact.scss

```diff
@@ -0,0 +1,140 @@
+.contact {
+  margin-bottom: 100px;
+  position: relative;
+
+  @include tablet {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 30px;
+    margin-bottom: 132px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    grid-gap: 120px;
+    margin-bottom: 132px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__title {
+    @include desktop {
+      margin-bottom: 30px;
+    }
+  }
+
+  &__image {
+    @include tablet {
+      max-width: 100%;
+      height: 0;
+      padding-bottom: 119%;
+      background: url(../images/bird.png) no-repeat right/contain;
+
+      @include desktop {
```

Comment:
The file path in CSS background url() is not recommended to be relative. It could lead to problems in finding the resource if the CSS file is moved. Consider using an absolute path or better a path relative to the root.